src/cascadia/TerminalSettingsModel/Profile.idl

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import "IAppearanceConfig.idl";
+import "AppearanceConfig.idl";

```

Comment:
we don't technically need this line, right?

Reply:
You're right! Removed